XenModel/XenAPI-Extensions/VM.cs

```diff
@@ -818,7 +830,9 @@ private VirtualisationStatus GetVirtualisationStatusNewVM
             {
                 Debug.Assert(HasNewVirtualisationStates);
 
-                var flags = HasRDP ? VirtualisationStatus.MANAGEMENT_INSTALLED : 0;
+                var flags = !Helpers.DundeeOrGreater(Connection) && HasRDP || HasStaticIP

```

Comment:
I think that DundeeOrGreater is not needed here, as this function will only be called for Dundee hosts